buildpackage.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Unless you're using Bash specific features, suggest making the shebang more portable.
```suggestion
#!/bin/sh
```